### PR TITLE
Change delete confirmation message

### PR DESCRIPTION
**Pull request by antoinebhs**
Change delete message to:
* always have "Confirmation" as title
* not display the list of items we delete (we only display the selected to-delete directory as the focus is not very clear now)
* add "permanently" to emphasize the fact that the deletion is definitive

### CODE REVIEW

File: src/components/dialogs/delete-dialog.js [hunk 1]
```diff
@@ -74,23 +74,8 @@ const DeleteDialog = ({
...
-                              tooltipSx={styles.tooltip}
-                          />
-                      ),
-                  }
-              )
-            : intl.formatMessage(
-                  { id: 'deleteMultipleItemsDialogTitle' },
-                  { itemsCount: items.length }
-              );
+    const buildTitle = () => {
```
Comment (TheMaskedTurtle) [inline src/components/dialogs/delete-dialog.js:77]: You can delete the argument in the method call

File: src/translations/fr.json [hunk 2]
```diff
@@ -65,12 +65,11 @@
   "renameFolder": "Renommer",
   "insertNewDirectoryDialogTitle": "Créer un dossier",
   "insertNewRootDirectoryDialogTitle": "Créer un dossier racine",
-  "deleteDirectoryDialogMessage": "Le dossier {itemName} ainsi que les éléments qu'il contient va être supprimé.",
-  "deleteMultipleDirectoriesDialogMessage": "{itemsCount} dossiers ainsi que les éléments qu'ils contiennent vont être supprimés.",
-  "deleteItemDialogMessage": "{itemName} va être supprimé.",
-  "deleteMultipleItemsDialogMessage": "{itemsCount} éléments vont être supprimés.",
-  "deleteItemDialogTitle": "Supprimer\u00a0{itemName}",
-  "deleteMultipleItemsDialogTitle": "Supprimer {itemsCount} éléments",
+  "deleteDirectoryDialogMessage": "Le dossier {itemName} ainsi que les éléments qu'il contient va être supprimé définitivement.",
```
Comment (TheMaskedTurtle) [inline src/translations/fr.json]: "vont être supprimés" ?

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: src/components/dialogs/delete-dialog.js (hunk 1 above)
Comment (antoinebhs) [reply src/components/dialogs/delete-dialog.js:77]: You're right... Fixed!

File: src/translations/fr.json (hunk 2 above)
Comment (antoinebhs) [reply src/translations/fr.json]: Fixed!

**Review by TheMaskedTurtle — APPROVED** [review]

**Pull request merged**